Reference/V9-Config/Serilog/index-v9.md

```diff
@@ -0,0 +1,79 @@
+---
+versionFrom: 9.0.0
+meta.Title: "Umbraco Serilog Settings"
+meta.Description: "Information on the serilog settings section"
+state: complete
+verified-against: beta-3
+update-links: true
+---
+
+# Serilog settings
+
+Umbraco uses Serilog as its logging library, this means that the configuration of logging is offloaded to Serilog instead of the CMS. This means that logging specific configuration is no the `Umbraco.Cms` section, but instead the Serilog section.
```

Comment:
```suggestion
Umbraco uses Serilog as its logging library, this means that the configuration of logging is offloaded to Serilog instead of the CMS. This means that logging specific configuration is not in the `Umbraco.Cms` section, but instead the Serilog section.
```